examples/high_level_functions_demos/types_for_members_demo.js

```diff
@@ -75,6 +75,69 @@ var memberStiffnessModification2 = new MemberStiffnessModification(undefined, st
 memberStiffnessModification2.SteelStructuresAisc(1, 2);
 var memberDefinableStiffness3 = new MemberStiffnessModification(undefined, structure_modification);
 memberDefinableStiffness3.SteelStructuresCSA(undefined, undefined, 1, 2);
+
+/********************************************************************************* Member opening ****************************************************************************/
+var member1 = new Member();
+member1.Beam(undefined, [102, 103], 1);
+var memberSet1 = new MemberSet();
+memberSet1.ContinuousMembers(undefined, [51, member1.GetNo(), 52]);
+
+// clearAll.js not removes member openings
+for (var i = member_openings.count(); i > 0; --i) {
+    member_openings.erase(member_openings.getNthObjectId(i));
+}
```

Comment:
```suggestion
for (var i = member_openings.count(); i > 0; --i) {
    member_openings.erase(member_openings.getNthObjectId(i));
}
```
This is not needed as I have added container member_opening to clearAll.js